README.md

```diff
@@ -1,8 +1,8 @@
-[![](https://raw.githubusercontent.com/mapbox/mapbox-java/master/.github/splash-img.png)](https://www.mapbox.com/android-docs/mapbox-services/)
```

Comment:
@cammace nit: can we change the icon for RxJava? Maybe simply an `X`, or an an arrow, to represent asynchronicity? Looks great otherwise!